pages/linux/po4a-gettextize.md

```diff
@@ -0,0 +1,16 @@
+# po4a-gettextize
+
+> Convert a file to a PO file.
```

Comment:
```suggestion
> Convert a text file to a PO file.
```
Otherwise it soudns really unspecific

Also I personally prefer `convert into a PO file`, but I think `to` is also fine. But in case you change it, please make sure to change it everywere

Reply:
po4a command can not only convert a text file but also other formats of files. Although it is a text file in this specific example, I think we should not use `text` file here.